helm-charts/falcon-sensor/README.md

```diff
@@ -78,6 +78,26 @@ The following tables lists the more common configurable parameters of the chart
 
 `falcon.cid` and `node.image.repository` are required values.
 
+## Container Sensor Configuration
+
+The following tables lists the more common configurable parameters of the chart and their default values for installing the Container sensor.
+
+| Parameter                                        | Description                                                             | Default                                   |
+|:-------------------------------------------------|:------------------------------------------------------------------------|:----------------------------------------- |
+| `container.enabled`                              | Enable installation on the Kubernetes node                              | `false`                                   |
+| `container.disableNSInjection`                   | Disable injection for all Namespaces                                    | `false`                                   |
+| `container.disablePodInjection`                  | Disable injection for all Pods                                          | `false`                                   |
+| `container.certExpiration`                       | Certificate validity duration in number of days                         | `3650`                                    |
```

Comment:
10 years? gtfo

Reply:
:: sigh :: or more appropriately, let's open an internal feature request to have this value reviewed.